src/navability/graphql/DataBlobs.jl

```diff
@@ -70,6 +70,50 @@ mutation sdk_adddataentry(\$userId: ResourceId!, \$robotId: ResourceId!, \$sessi
 }
 """
 
+GQL_ADDBLOBENTRY = """
+mutation sdk_addblobentry(
+  \$userId: String!
+  \$robotId: String!
+  \$sessionId: String!
+  \$variableLabel: String!
+  \$dataId: UUID!
```

Comment:
not sure if this should be `blobId`?  Not familiar with the latest here yet.  In the old system this used to be `fileId`

Reply:
I think you are correct, I just copied the DataEntry one.